lib/database/context/permission.ex

```diff
@@ -97,4 +97,31 @@ defmodule BorsNG.Database.Context.Permission do
       patch: patch
     })
   end
+
+  def undelegate(user_id, patch_id) do
+    Repo.delete_all(
+      from(d in UserPatchDelegation,
+        where:
+          d.user_id == ^user_id and d.patch_id == ^patch_id
+      )
+    )
+  end
+
+  def undelegate_patch(patch_id) do
+    Repo.delete_all(
+      from(d in UserPatchDelegation,
+        where:
+          d.patch_id == ^patch_id
+      )
+    )
+  end
+
+  def undelegate_user(user_id) do
+    Repo.delete_all(
+      from(d in UserPatchDelegation,
+        where:
+          d.user_id == ^user_id
+      )
+    )
+  end
```

Comment:
`undelegate(user_id, patch_id)` and `undelegate_user(user_id)` are unused in this PR but might be useful for future features: undelegating a specific user from a specific PR and undelegating a user from all PRs, respectively